code/game/objects/structures/arclumin.dm

```diff
@@ -0,0 +1,21 @@
+/*
+
+At 7:30 AM on March 8th, 2017, ArcLumin died in a fatal car crash on impact.
```

Comment:
What's the good in pretending that Arc's death WASN'T what prompted this PR? Leave it in for the sake of history.